### ISSUE TITLE
NPM package does not match Github

### ISSUE THREAD

**Issue by weavst** [driverdan/node-XMLHttpRequest#182]
There are several changes that are not updated in the npm version, including the bug fixes for "getRequestHeader" and the IPv6 issue.

Could you please publish the latest version of this package to npm?

**Comment by TheDude53** [comment]
I agree, the NPM package should be updated to match the repo. Until then, it looks like we'll have to use `npm install https://github.com/driverdan/node-XMLHttpRequest/tarball/master` instead.
